src/controller/index.rs

```diff
@@ -1,5 +1,17 @@
 use actix_web::{HttpResponse, Responder};
+use askama::Template;
+
+#[derive(Template)]
+#[template(path = "index.html")]
+
+struct IndexTemplate<'a> {
+    name: &'a str,
```

Comment:
[IMO]
strでは固定サイズかつUTF-8は保証されないため、標準ライブラリの提供するStringを使った方が良いです。
https://qiita.com/yagince/items/e7474839246ced595f7a
```suggestion
    name: &'a String,
```

Reply:
Stringを使うのであればはいらないと思いますね。